### PR TITLE
V5 ga 246 sso configuration

### PR DESCRIPTION
**Pull request by SrdjanReblaze**
image

### CODE REVIEW

File: src/components/FormInput.vue [hunk 1]
```diff
@@ -19,20 +19,15 @@
...
            :disabled="disabled"
-           @input="$emit('update:inputValue', ($event.target as HTMLInputElement).value)"/>
-    <span v-if="icon && iconLocation"
-          class="material-symbols-outlined icon"
-          :class="{
-            'is-left': iconLocation === 'left',
-            'is-right': iconLocation === 'right'
-          }">
-      {{ icon }}
-    </span>
```
Comment (Aviv-Galmidi) [inline src/components/FormInput.vue:35]: This is needed, please restore the icon code

File: src/views/integrations/SsoSettings.vue [hunk 2]
```diff
@@ -107,15 +86,13 @@
...
+                    <form-input name="idp-group-input"
+                                dataQa="idp-group-input"
+                                title="Add IDP group claim"
+                                placeholder="IDP group claim"
+                                :disabled="!ssoData.enabled"
+                                :inputValue="role.idp_group"
+                                @update:inputValue="role.idp_group = $event"/>
                   </div>
                   <div class="button-wrapper-column column">
                     <button
```
Comment (Aviv-Galmidi) [inline src/views/integrations/SsoSettings.vue:98]: Button was not converted

Screen Shot 2023-05-04 at 10 25 19

File: vue.config.js [hunk 3]
```diff
@@ -8,11 +8,11 @@ module.exports = {
     proxy: {
       '^/conf/api': {
         pathRewrite: {'^/conf/api': '/api'},
-        target: 'http://127.0.0.1:5000',
+        target: 'http://curie.demo:30000',
       },
       '^/reblaze/api': {
         pathRewrite: {'^/reblaze/api': '/api'},
-        target: 'http://127.0.0.1:8000',
+        target: 'http://35.234.67.139:8911',
```
Comment (Aviv-Galmidi) [inline vue.config.js:15]: Please revert to original values

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
1. Labels do not match one another in the page, some are bold and some are not

Screen Shot 2023-05-04 at 10 19 56

**Comment by SrdjanReblaze** [comment]
> 1. Labels do not match one another in the page, some are bold and some are not
> 
> Screen Shot 2023-05-04 at 10 19 56

Yes, because on figma design there is no label for Dropdown fields, so i leave labels that was there, they are bolder

File: src/components/FormInput.vue (hunk 1 above)
Comment (SrdjanReblaze) [reply src/components/FormInput.vue:35]: done

File: vue.config.js (hunk 3 above)
Comment (SrdjanReblaze) [reply vue.config.js:15]: done

File: src/views/integrations/SsoSettings.vue (hunk 2 above)
Comment (SrdjanReblaze) [reply src/views/integrations/SsoSettings.vue:98]: image

done

**Review by Aviv-Galmidi — DISMISSED** [review]
Please fix the conflicts and then we can merge

**Comment by SrdjanReblaze** [comment]
> Please fix the conflicts and then we can merge

done

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**